Library/CompilerFile.php

```diff
@@ -573,11 +574,6 @@ public function preCompile(Compiler $compiler)
         }
 
         /**
-         * Alias Manager
-         */
-        $this->_aliasManager = new AliasManager();
```

Comment:
I have change the alias manager instantiation just for not to be force to call "preCompile" method